index.html

```diff
@@ -72,6 +72,9 @@ <h1 class="title">HERE Maps on GitHub</h1>
 				client_secret : '918cbdf3a49ba292ff5e51e01e893ae0117c93ac'
 				},
 				repos = [
+					{id: 'mobility-on-demand-use-cases-ios',
+					type: 'Objective-C',
+					description:'HERE mobility on-demand demo written as an iOS application'},
```

Comment:
Hmm, to be frank that argument does not really convince me / seem sound to me, but it's probably not worth blocking the PR because of that. Let's revisit this when we put the descriptions in the respective project settings and retrieve them via the API instead of hard-coding / duplicating them here.